### ISSUE TITLE
[Bug]: Build: Xcode 16 Beta requires Sentry 8.29.0 and Lottie 4.4.3

### ISSUE THREAD

**Issue by shinra-electric** [cryptoAlgorithm/Swiftcord#190]
### Describe the Bug

**Bug Description**
Building fails in Xcode 16 beta due to Sentry and Lottie.
A fix for Sentry has been merged in `8.29.0`
I'm unsure which version of Lottie fixes its issue, but building succeeds when setting to latest (`4.4.3`) 

### Reproducing the Bug

Try to build the project with macOS 15 Sequoia beta and Xcode 16 beta

### Version

0.7.1

### Category

Other

**Comment by DeadstarIII** [comment]
yeah same here, can't run it nor build it, tagged as malware and build doesn't work
